includes/class-llms-events-query.php

```diff
@@ -132,11 +132,10 @@ protected function parse_args() {
 	}
 
 	/**
-	 * Prepare the SQL for the query
+	 * Prepare the SQL for the query.
 	 *
 	 * @since 3.36.0
 	 * @since 4.7.0 Use `$this->sql_select_columns({columns})` to determine the columns to select.
-	 *
```

Comment:
This should stay, we probably don't have a cs rule for this but I think we should.
We always left on "blank" line before the `@param`, or before `@return` when there are no `@param`.
@thomasplevy what do you think?

Reply:
I haven't taken the time to write a PHPCS rule for this but it is in our doc standards ( and I think the WP core's too )

Reply:
At least it *should* be in our doc standards. If it's not it should be since 99% of our code adheres to it (intentionally)